### ISSUE TITLE
1.5 Hours PPP

### ISSUE THREAD

**Issue by anish-kum** [anish-kum/PPP-Spark-2025-2026#23]
90 min PPP

**Comment by anish-kum** [comment]
<img width="1277" height="552" alt="Image" src="https://github.com/user-attachments/assets/4940c180-4976-43ab-a8d1-fbe4121ddf5b" />

<img width="318" height="48" alt="Image" src="https://github.com/user-attachments/assets/d4bb3ee3-d4c9-49d5-84f6-0a5bb70d1564" />

---------------------------------------------------------------------------------------------------------------------------------------
What I learned:

- Convolutional Kernels: Discusses the role of convolutional kernels in extracting features from input images during the convolution operation.

- Filter Visualization: Explains techniques to visualize filters and understand what features they are detecting in the input data.

- Activation Maps: Introduces activation maps, which show the output of convolutional layers and highlight important features learned by the model.

- Pooling Layers: Covers the purpose of pooling layers in reducing the spatial dimensions of feature maps while retaining essential information.

- Transfer Learning: Discusses how pre-trained models can be fine-tuned for specific tasks, leveraging learned features from large datasets.

- Data Augmentation: Explains techniques to expand the training dataset artificially, improving model robustness and generalization.

- Model Evaluation: Emphasizes the importance of evaluating model performance using metrics like accuracy, precision, and recall.ayers: Describes how fully connected layers are used after convolutional and pooling layers to make final predictions.

---------------------------------------------------------------------------------------------------------------------------------------

2 Questions I still had:
-How does the choice of convolutional kernel size and number affect what types of features (edges, textures, shapes) a CNN can learn at different layers?

-When using transfer learning, how do you decide which layers to freeze and which to fine-tune, and how does that choice impact overfitting and model performance on a new dataset?

**Comment by drewmca-pkwy** [comment]
@anish-kum Anish, this looks good. Clear learnings, screenshot of progress, and questions. I'm applying these 90 minutes to last week's turn-in (1.16).
